index.rst

```diff
@@ -69,18 +52,16 @@ in the :doc:`Encrypted Email Overview <encrypted_email_overview>`.
 
    general_support
 
-Motivation
-----------
+-----
+
+First Time Contacting Us?
+-------------------------
 
-Our goal with this new support system is to simplify and centralize the
-process of providing support to the administrators who maintain and the
-journalists who use SecureDrop. Administering a SecureDrop instance is
-hard work, and we hope that this new system will make us more effective
-in providing help to you.
+If this is your first time contacting FPF for assistance, or if you have
+questions about
 
-.. note:: Don't hesitate to let us know if you have any feedback on the
-          new support system.
+* installing SecureDrop;
+* an existing SecureDrop installation that was set up without our involvement; or
+* inclusion in the `SecureDrop Directory <https://securedrop.org/directory>`_;
 
-          If you already have an account on the support site, create a new issue
-          in your project to provide feedback. Otherwise, use the
-          :doc:`general_support` channel.
+please contact us through the :doc:`general_support` channel.
```

Comment:
While you're at it, it's definitely worth adding a reference to https://forum.securedrop.org/c/support for public support inquiries here, but this can also be handled in follow-up.